### ISSUE TITLE
NTR: Autonomic status epilepticus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5349]
"-Non-convulsive status epilepticus without coma
---Focal non-convulsive status epilepticus without coma
---Generalized non-convulsive status epilepticus without coma
---NTR: Autonomic status epilepticus
label: Autonomic status epilepticus
def: Autonomic status epilepticus is a type of non-convulsive status epilepticus without coma with prominent autonomic features regardless of whether it is electrographically generalized or focal.
comment: Autonomic disturbances may include tachycardia, cardiac arrhythmia, hypertension, high fever, salivation, vomiting and incontinence.
source: PMID: 26336950
http://webprotege.stanford.edu/RiCnLhdRh6VgSt0KNJ92ib: ILAE 2015
http://www.w3.org/2000/01/rdf-schema#current_hpo: nan
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney"
PMID:28276060: https://www.ncbi.nlm.nih.gov/pubmed/PMID:28276060

**Comment by pnrobinson** [comment]
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney


Should this term also be a child of https://hpo.jax.org/app/browse/term/HP:0011154

**Comment by DLewisSmith** [comment]
Not once the new focal seizure subontology is in place. There is a conceptual difference. The current status Epilepticus & old focal seizure classifications definitions were usually interpreted to classify seizures based on their MOST PROMINENT features. The new focal seizure classification does so by the EARLIEST discernible feature other than momentary behavioural arrest ('Unitary focal seizures are named for the initial manifestation and presence or absence of altered consciousness at any point during the seizure’).

Thus autonomic SE is not necessary a type of prolonged focal seizure with autonomic features, because autonomic SE may have begun with another feature (e.g. jerks of an arm).


On 22 Feb 2020, at 17:12, Peter Robinson <notifications@github.com<mailto:notifications@github.com>> wrote:

﻿

@DLewisSmith<https://github.com/DLewisSmith>
@galerp<https://github.com/galerp>
@shiva-g<https://github.com/shiva-g>
@ihelbig<https://github.com/ihelbig>
@GBalagura<https://github.com/GBalagura>
@hughkearney<https://github.com/hughkearney>

Should this term also be a child of https://hpo.jax.org/app/browse/term/HP:0011154

**Comment by pnrobinson** [comment]
finished adding term.
